week1/index.js

```diff
@@ -1,38 +1,30 @@
-import HTTP from 'http'
-import Path from 'path'
-
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
-
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
-
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
-	}
+const http = require('http');
+const port = 3000;
+let state = 10;
+let server = http.createServer((request, response)=>{
```

Comment:
Your spacing and formatting is okay (though I prefer a different style), but at least you were consistent with your code. However, the code you copied from me 😛 on line 29 is different - you should try and make sure that your style and formatting is consistent within a file.

For example, on line 4, you write `http.createServer((request, response)=>{` (ie. without spaces around the arrow or brackets).
I wrote `server.listen(port, () => {` on line 29, with spaces around the arrows.

I'd recommend you use spaces around the arrows and curly braces (like, for example, `} else if (request.url === "/remove") {` for line 11), as I think it makes it more readable and is also more consistent with JavaScript conventions. But whichever style you choose, make sure it's always the same.